.github/workflows/intellij_plugin_verification.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   intellij_plugin_verification:
 
     env:
-      INTELLIJ_PLUGIN_VERIFIER: 1.241
+      INTELLIJ_PLUGIN_VERIFIER: 1.249
```

Comment:
I'm going to create a pull request to remove this value. It's already in `gradle.properties` :+1:

Reply:
Though the current failure doesn't come from the Intellij Plugin Verifier tool. It's not finding the compatibility versions properly. I'm looking at it now :raised_hands:

Reply:
The error comes from this file: https://github.com/JetBrains/kotlin/blob/v1.4.10/idea/resources/META-INF/plugin.xml.202

```
  <idea-version since-build="202.1" until-build="203.*"/>
```

and 2020.3 doesn't exist yet!

I think it's an error from their side. I'm going to review the best way to fix it...

Reply:
I also found another issue
It's now required to set the JAVA_HOME, to run the verifier at least locally. 
I got it working with setting 
```
runPluginVerifier {
  pluginFileName = "Arrow Meta-$VERSION_NAME"
  ides = ["IC-$INTELLIJ_IDEA_VERSION_UNDER_TEST"]
  verifierVersion = "$INTELLIJ_PLUGIN_VERIFIER"
  runtimeDir = "/Library/Java/JavaVirtualMachines/jdk1.8.0_201.jdk/Contents/Home"
}
```

Reply:
Yes, it's necessary JAVA_HOME or indicating the `runtimeDir` in the way you did. `JAVA_HOME` is already created in GitHub environments :+1:  BTW, I collaborated with an issue in that sense: https://github.com/JetBrains/intellij-plugin-verifier/pull/45